dht/token.hh

```diff
@@ -39,9 +42,9 @@ enum class token_kind {
 class token_view {
```

Comment:
Maybe it'd make more sense to just remove the `token_view` class now. The reason this class exists is that it's trivially destructible, so it's easy to dispose giant vectors holding `token_view`s (https://github.com/scylladb/scylla/commit/dddc7e1676972a3acd34f157b4d430593c4b4099).
But now `token` is trivially destructible. (I'd suggest to add a `static_assert` to check this)

Reply:
Oh, nevermind, I see you do it in a later commit.
https://github.com/scylladb/scylla/pull/5676/commits/bb240792a92004cd726e748a272d2bc960ffd9a6

Reply:
It's better not to mix code movement with changes.

Reply:
Huh, github shows me this conversation on every commit that touches this file. Thanks github!

Reply:
Hey, from yet another commit!